webpack.config.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+module.exports = {
+  module: {
```

Comment:
- [ ] The 'mode' option has not been set. Kindly set up your Webpack for development or production. You can check this [link](https://webpack.js.org/guides/production/) out.